env0/data_sshkey_test.go

```diff
@@ -0,0 +1,67 @@
+package env0
+
+import (
+	"github.com/env0/terraform-provider-env0/client"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"testing"
+)
+
+func TestUnitSshKeyDataSourceById(t *testing.T) {
```

Comment:
It really feels like we can DRY those two test functions. I think it's worth an effort 🙏

Reply:
a1a0f56

Reply:
![](https://media0.giphy.com/media/ZdUnQS4AXEl1AERdil/giphy.gif)